cmslite_metadata.view.lkml

```diff
@@ -12,4 +12,14 @@ view: cmslite_metadata {
   dimension: title {}
   dimension: hr_url {}
   dimension: language {}
+  dimension: page_language {
+    type: string
+    sql: ${TABLE}.language_name ;;
+    description: "The language specified in the CMS Lite page settings tab"
+  }
+  dimension: language_code {
+    type: string
+    sql: ${TABLE}.language_name ;;
```

Comment:
Is the language code actually specified, or is it just a key associated to the language specified?

Reply:
It isnt specified in page settings in cmslite, so it appears to just be a key associated to the language value.